integration-tests/src/test/java/tachyon/underfs/s3/S3UnderStorageCluster.java

```diff
@@ -38,15 +39,14 @@
   public S3UnderStorageCluster(String baseDir, TachyonConf tachyonConf) {
     super(baseDir, tachyonConf);
     mS3Bucket = System.getProperty(INTEGRATION_S3_BUCKET);
-    mBaseDir = mS3Bucket + UUID.randomUUID();
+    mBaseDir = PathUtils.concatPath(mS3Bucket + UUID.randomUUID());
```

Comment:
You don't need to pass both components to concatPath()? 2 components instead of 1?